.test-defs/cmd/create-shoot/main.go

```diff
@@ -135,6 +144,19 @@ func main() {
 	updateMachineType(shootObject, cloudprovider, machineType)
 	updateAutoscalerMinMax(shootObject, cloudprovider, autoScalerMin, autoScalerMax)
 	updateFloatingPoolName(shootObject, floatingPoolName, cloudprovider)
+	if dnsProvider != "" {
+		shootObject.Spec.DNS.Provider = dnsProvider
+	}
+	if hostedZoneID != "" {
+		shootObject.Spec.DNS.HostedZoneID = &hostedZoneID
```

Comment:
This will be deprecated and removed with #850. Why do you need it now? What are you testing with DNS? Can we postpone this change until #850 is merged?

Reply:
Currently used it for testing a gardener deployed with sow. But I rechecked with @Diaphteiros that we do not need it anymore with the new DNS controller.
Therefore, I will remove this parameter.